src/main/java/org/myrobotlab/service/config/ServoConfig.java

```diff
@@ -2,18 +2,18 @@
 
 public class ServoConfig extends ServiceConfig {
 
-  public boolean autoDisable;
+  public boolean autoDisable = false;
```

Comment:
not related to this PR.. this should have been done on develop... bad grog!

Reply:
heh ... hotfix fallout ..
the clip boolean wasn't set ...
come on 1 line ! ;) ... can I hotfix it too ?  if i back it out I'll forget to explicitly set